### PR TITLE
Updating document to reflect current technical state.

### PR DESCRIPTION
**Pull request by KoprowskiT**
Updates parts about:
- reserved instances
- azure hybrid benefit plus a few minor changes

### CODE REVIEW

**Comment by KoprowskiT** [comment]
Comments on why changes are requested:
Line: 31 / As you are using short names, and later SQL VM is known as an acronym, it is good to add it here
Line 61 / We do not have options 1-3, like 1 or 2 or 3 for reserved instances. We can choose only 1 year OR 3 years term. We have a five-year reservation available for VM, but it is only for HB v 2 family and only in South Central US and North Europe - not sure if it's worth mentioning it here.
Line: 79 / We can have readers who do not know the ESG acronym
Line: 153 / We can use AHB in five ways: Azure Iaas / Azure Data Factory / Azure SQL DB / SQL Managed Instances / Azure Arch MI - I think it is important to update documentation plus add a link to the licensing description. (https://cloudblogs.microsoft.com/sqlserver/2023/04/26/capitalize-on-your-investments-with-the-new-centrally-managed-azure-hybrid-benefit-for-sql-server/)
Line: 257 / Acronym not explained before, so adding it here 

+ two of three small typos.

**Comment by Court72** [comment]
@rwestMSFT

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/migration-guides/modernization.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ A [migration to Azure](https://azure.microsoft.com/overview/cloud-migration-bene
 
 #### What is Azure SQL?
 
-[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml).
+[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml) (SQL VM).
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md:31]: In a lot of cases, we aren't allowed to abbreviate service names. This is one such example.

```suggestion
[Azure SQL](../azure-sql-iaas-vs-paas-what-is-overview.md) is a family of services that use the SQL Server database engine in the Azure Cloud. The following services belong to Azure SQL: [Azure SQL Database](../database/sql-database-paas-overview.md) (SQL Database), [Azure SQL Managed Instance](../managed-instance/sql-managed-instance-paas-overview.md) (SQL Managed Instance) and [SQL Server on Azure VMs](../virtual-machines/index.yml).
```

File: azure-sql/migration-guides/modernization.md [hunk 2]
```diff
@@ -282,11 +282,11 @@ See the [Performance](/azure/azure-sql/managed-instance/frequently-asked-questio
...
 
 #### Where can I find migration guides to Azure SQL?
 
-The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/) that also contains guides for migrating to another database targets.
+The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/) which also contains guides for migrating to another database targets.
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md]: In our style guide, "which" always takes a preceding comma, whereas "that" doesn't.

```suggestion
The following guides help you discover, assess, and migrate from SQL Server to [SQL VM](./virtual-machines/sql-server-to-sql-on-azure-vm-individual-databases-guide.md), [SQL Managed Instance](./managed-instance/sql-server-to-managed-instance-overview.md) and [SQL Database](./database/sql-server-to-sql-database-overview.md). You can consult [Azure Database Migration Guides](/data-migration/), which also contains guides for migrating to another database targets.
```

File: azure-sql/migration-guides/modernization.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ SQL Server CAL licenses with appropriate [license mobility](https://www.microsof
...
+Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and/or SQL Server licenses (both Standard and Enterprise) with Software Assurance (SA) to Azure IaaS & Paas services. If you are planning to bring only existing SQL Server licenses, Azure Hybrid Benefits allow you bringing licences to Azure Data Factory SQL Server Integration Services, Azure SQL Database (both Elastic Pool and Single Database), SQL Managed Instances (General Purpose, Business Critical and Hyperscale) and Azure Arc-enabled SQL Managed Instance (both General Purpose and Business Critical). AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates. Azure Hybrid Benefit is not available in the serverless compute tier of Azure SQL Database. For more detailed information, how to use AHB with your own licenses see [Microsoft Azure Services Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftAzureServices/EAEAS)
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md]: We generally do not discuss licensing details in Docs. This change will likely not be approved.

```suggestion
Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows you bringing your existing Windows Server and SQL Server licenses with Software Assurance (SA) to Azure. AHB can bring you up to 85% savings compared to pay-as-you-go pricing in Azure SQL, when combined with reservations savings and extended security updates. For more information, see [Microsoft Azure Services Product Terms](https://www.microsoft.com/licensing/terms/productoffering/MicrosoftAzureServices/EAEAS).
```

File: azure-sql/migration-guides/modernization.md [hunk 4]
```diff
@@ -68,15 +68,15 @@ Unique to Azure, Azure Hybrid Benefit (AHB) is a licensing benefit that allows y
...
+If you're interested to understand how you can save up to 64% by moving to Azure SQL please check ESG (Environmental, Social and Governance Impact report) report on [The Economic Value of Migrating On-Premises SQL Server Instances to Microsoft Azure SQL Solutions](https://azure.microsoft.com/resources/the-economic-value-of-migrating-on-premises-sql-server-instances-to-microsoft-azure-sql-solutions/). Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md]:
```suggestion
If you would like to understand how you can save up to 64% by moving to Azure SQL please check the Environmental, Social and Governance Impact (ESG) report on [The Economic Value of Migrating On-Premises SQL Server Instances to Microsoft Azure SQL Solutions](https://azure.microsoft.com/resources/the-economic-value-of-migrating-on-premises-sql-server-instances-to-microsoft-azure-sql-solutions/). Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```

File: azure-sql/migration-guides/modernization.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ Moving to Azure brings savings in resource, maintenance, and real estate costs,
...
 
-In addition, every Azure SQL service provides you the option to reserve instances in advance (1-3 years) and obtain significant additional savings. [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest) provide a way to further reduce development costs. Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
+In addition, every Azure SQL service provides you the option to reserve instances in advance (for 1 year or 3 years term) and obtain significant additional savings. [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest) provide a way to further reduce development costs. Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md]:
```suggestion
In addition, every Azure SQL service provides you the option to reserve instances in advance (for terms of either 1 year or 3 years) and obtain significant additional savings. [Dev/Test pricing plans](https://azure.microsoft.com/pricing/dev-test/#devtest) provide a way to further reduce development costs. Finally, check the following article on how you can [Optimize your Azure SQL Managed Instance cost with Microsoft Azure Well-Architected Framework](https://techcommunity.microsoft.com/t5/azure-sql-blog/optimize-your-azure-sql-managed-instance-cost-with-microsoft/ba-p/2235216).
```

File: azure-sql/migration-guides/modernization.md [hunk 6]
```diff
@@ -282,11 +282,11 @@ See the [Performance](/azure/azure-sql/managed-instance/frequently-asked-questio
 
 #### I want to modernize SQL Server workloads to Azure SQL. What is the next step?
 
-A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated Migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding Migration team required skill set and overall process.
+A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated Migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding the Migration team required skill set and overall process.
```
Comment (rwestMSFT) [inline azure-sql/migration-guides/modernization.md]:
```suggestion
A great place to start is joining the [Azure Migration and Modernization Program](https://azure.microsoft.com/migration/migration-modernization-program/#overview). When you start a migration project, a good practice is to form a dedicated migration team to formulate and execute the migration plan. If your company has an assigned Microsoft or Microsoft Partner account team, they can provide guidance regarding the migration team required skill set and overall process.
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]

File: azure-sql/migration-guides/modernization.md (hunk 1 above)
Comment (KoprowskiT) [reply azure-sql/migration-guides/modernization.md:31]: But you are using SQL VM later in the same article. few times like in lines 45, 59, 75, 77, 91, 105, 149, 155, 159, and none of those abbreviations have been written by me. I just see it logical to add this shortened form at the beginning of the article for better understanding.

File: azure-sql/migration-guides/modernization.md (hunk 3 above)
Comment (KoprowskiT) [reply azure-sql/migration-guides/modernization.md]: I did not add any extras about licensing in detail; I just updated options for AHB. Understand rejection. It is better not to say anything than to present only part of the information. How about the link for details (my last sentence) for easy navigation?

**Review by KoprowskiT — COMMENTED** [review]
A few comments were added to the discussion.

File: azure-sql/migration-guides/modernization.md (hunk 1 above)
Comment (rwestMSFT) [reply azure-sql/migration-guides/modernization.md:31]: They shouldn't be there then, so I'll have to remove all references to that. That's part of the excitement of inheriting content.

Comment (KoprowskiT) [reply azure-sql/migration-guides/modernization.md:31]: Happy to help then :)

**Comment by rwestMSFT** [comment]
Your changes will be incorporated into an internal PR, which will be published in the next few business days. When that PR is merged, this one will close automatically.

Thank you again for your ongoing contributions.

**Pull request closed without merging**